packages/botkit/src/css/botkit.css

```
@@ -0,0 +1,880 @@
+.bk-back:hover {
+  color: var(--bk-ink);
+  border-color: var(--bk-accent-line);
+}
+
+.bk-page-title {
+  margin: 0;
+  font-size: var(--bk-text-xl);
+  font-weight: 700;
+  line-height: 1.2;
+  letter-spacing: -0.01em;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)